### PR TITLE
add source features to update site

### PR DESCRIPTION
**Pull request by majuwa**
Relates [to](https://github.com/PalladioSimulator/Palladio-Addons-DataFlowConfidentiality-Analysis/issues/32)

### CODE REVIEW

**Comment by Entenwilli** [comment]
A smaller update on the issue:
The workflow fails, due to an issue resolving the referenced genmodel: In debugging Eclipse is not able to find the following genmodels:
- `platform:/resource/org.palladiosimulator.pcm/model/pcm.genmodel#//pcm`
- `platform:/resource/de.uka.ipd.sdq.probfunction/model/ProbabilityFunction.genmodel#//probfunction`
- `platform:/resource/de.uka.ipd.sdq.stoex/model/stoex.genmodel#//stoex`
- `platform:/resource/de.uka.ipd.sdq.units/model/Units.genmodel#//units`

**Comment by Entenwilli** [comment]
The pipeline should be fixed now, multiple projects for the Node Characteristics had an incorrect project name and the workflow was incorrectly configured. However, the pipeline should build the artifacts correctly again. @majuwa @sebinside

**Comment by Nicolas-Boltz** [comment]
@Entenwilli Could you change the runner-label in the build file to our self-hosted runners? Just so we can rule out any unexpected behavior.

**Comment by Entenwilli** [comment]
@Nicolas-Boltz I changed the runner label to `self-hosted` so it should run on the self-hosted runners

File: bundles/org.palladiosimulator.dataflow.nodecharacteristics.tests/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %pluginName
+Bundle-SymbolicName: org.palladiosimulator.dataflow.nodecharacteristics.tests;singleton:=true
+Automatic-Module-Name: org.palladiosimulator.dataflow.nodecharacteristics.tests
+Bundle-Version: 1.0.0.qualifier
+Bundle-ClassPath: .
+Bundle-Vendor: %providerName
+Bundle-Localization: plugin
+Bundle-RequiredExecutionEnvironment: J2SE-1.5
```
Comment (Nicolas-Boltz) [inline bundles/org.palladiosimulator.dataflow.nodecharacteristics.tests/META-INF/MANIFEST.MF]: Change to `JavaSE-17`

File: bundles/org.palladiosimulator.dataflow.nodecharacteristics/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -7,7 +7,7 @@ Bundle-Version: 0.1.0.qualifier
 Bundle-ClassPath: .
 Bundle-Vendor: %providerName
 Bundle-Localization: plugin
-Bundle-RequiredExecutionEnvironment: JavaSE-11
+Bundle-RequiredExecutionEnvironment: J2SE-1.5
```
Comment (Nicolas-Boltz) [inline bundles/org.palladiosimulator.dataflow.nodecharacteristics/META-INF/MANIFEST.MF]: Change to `JavaSE-17`

**Review by Nicolas-Boltz — CHANGES_REQUESTED** [review]
Just two minor changes, everything else looks good.

File: bundles/org.palladiosimulator.dataflow.nodecharacteristics.tests/META-INF/MANIFEST.MF (hunk 1 above)
Comment (Entenwilli) [reply bundles/org.palladiosimulator.dataflow.nodecharacteristics.tests/META-INF/MANIFEST.MF]: Should be resolved by the most recent commit

File: bundles/org.palladiosimulator.dataflow.nodecharacteristics/META-INF/MANIFEST.MF (hunk 2 above)
Comment (Entenwilli) [reply bundles/org.palladiosimulator.dataflow.nodecharacteristics/META-INF/MANIFEST.MF]: Should be resolved by the most recent commit

**Review by Nicolas-Boltz — APPROVED** [review]
LGTM

**Pull request merged**